### DISCUSSION TITLE
Weather not working anymore

### DISCUSSION THREAD

**Discussion by Bad3m3ist3r** [matt8707/ha-fusion#438] [General]
Hi,

Since I updated to the latest version, my weather forecast is not working anymore. I tried different entities but no luck here. Also I tried different settings like hour or daily forecast ... It's just like that:

![CleanShot 2024-04-21 at 23 11 33@2x](https://github.com/matt8707/ha-fusion/assets/157047741/9de70140-8377-4415-91f7-024483386e53)

Does anyone have the same? What can I do?

Cheers

**Comment by matt8707** [comment]
Have you updated to the latest HA? The way forecasts work has changed.

**Comment by Bad3m3ist3r** [comment]
Yes, I am on version the following versions:

Core 2024.4.3
Supervisor 2024.04.0
Operating System 12.2
Frontend 20240404.2

still not working here.

**Comment by matt8707** [comment]
Issue https://github.com/matt8707/ha-fusion/issues/443
